src/astt_gui/templates/index.html

```diff
@@ -182,27 +182,12 @@ <h4>Unstow Before Tracking</h4>
         <ul id="components-nav" class="nav-content collapse " data-bs-parent="#sidebar-nav">
           <li>
             <a href="#">
-              <i class="bi bi-circle"></i><span>2020</span>
-            </a>
-          </li>
-          <li>
-            <a href="#">
-              <i class="bi bi-circle"></i><span>2021</span>
-            </a>
-          </li>
-          <li>
-            <a href="#">
-              <i class="bi bi-circle"></i><span>2022</span>
-            </a>
-          </li>
-          <li>
-            <a href="#">
-              <i class="bi bi-circle"></i><span>2023</span>
+              <i class="bi bi-circle"></i><span>2024</span>
```

Comment:
What will happen if the software is run in 2025 or 2026 or later? Should maybe say "latest" instead of putting a year?

Reply:
I think latest seems likely to be the best option.